ant/src/utils.lua

```diff
@@ -0,0 +1,39 @@
+local constants = require(".constants")
+local utils = {}
+
+function utils.hasMatchingTag(tag, value)
+	return Handlers.utils.hasMatchingTag(tag, value)
+end
+
+function utils.reply(msg)
```

Comment:
What kind of messages is this passing back?  Its unclear how this utils.reply function is doing.  For example, here is the current getRecord on the existing ANT experimentation script.

```
Handlers.add("getRecord", Handlers.utils.hasMatchingTag("Action", "Get-Record"), function(msg)
    if msg.Tags.SubDomain and Records[msg.Tags.SubDomain] then
        ao.send({
            Target = msg.From,
            Tags = {
                Action = "Record-Resolved",
                SubDomain = msg.Tags.SubDomain,
                TransactionId = Records[msg.Tags.SubDomain].transactionId,
                TtlSeconds = tostring(Records[msg.Tags.SubDomain].ttlSeconds),
            },
        })
    elseif Records["@"] then -- If no SubDomain is provided, then return the root subdomain
        ao.send({
            Target = msg.From,
            Tags = {
                Action = "Record-Resolved",
                SubDomain = "@",
                TransactionId = Records["@"].transactionId,
                TtlSeconds = tostring(Records["@"].ttlSeconds),
            },
        })
    else
        ao.send({
            Target = msg.From,
            Tags = { Action = "Get-Record-Error", ["Message-Id"] = msg.Id, Error = "Requested non-existant record" },
        })
    end
end)
```

Reply:
utils.reply is a wrapper around Handlers.utils.reply https://github.com/permaweb/aos/blob/f9036464f66452a73b4391da35909322df117b38/process/handlers-utils.lua#L36

If a string is passed in, then its sent to the msg.From address with the string in the data.

If a table is passed in, it treats the table as Tags to pass back.